packages/backend/src/database/entities/spaces.ts

```diff
@@ -152,6 +153,7 @@ export const getSpaceWithQueries = async (
                 chart_config: ChartConfig['config'];
                 chart_type: ChartType;
                 views: string;
+                first_viewed_at: string | null;
```

Comment:
why not using `date | null` instead ?

Reply:
db query returns a string or am I missing something?